bin/sirius-script-mamba-env-create.bash

```diff
@@ -0,0 +1,580 @@
+$COMM pyepics=3.5.0 pcaspy==0.7.3 pydm=1.10.3 timechart=1.2.3
+# remove the activate and deactivate files created by pyepics and pydm:
+cd $CONDA_PREFIX/etc/conda/activate.d
+if [ -f "pydm.bat" ]
+then
+    rm pydm.bat pydm.sh pyepics_activate.sh epics_base.sh
+fi
+cd $CONDA_PREFIX/etc/conda/deactivate.d
+if [ -f "pydm.sh" ]
+then
+    rm pydm.sh pyepics_deactivate.sh epics_base.sh
+fi
+
+printf_yellow_clear "- Install and configure jupyter notebook\n"
+$COMM jupyter notebook
+mamba update jupyter_client
+$COMM jupyter_contrib_nbextensions
+
+### Clone and install our repositories
+if [ "$CLONE" == "yes" ]
+then
+    printf_yellow "Clone and install our applications.\n"
+    printf_yellow_clear "Creating folder $CONDA_PREFIX/repos: "
+    if ! [ -d $CONDA_PREFIX/repos ]
+    then
+        mkdir $CONDA_PREFIX/repos
+        printf_green "done!\n"
+    else
+        printf_blue "already exists. Skipping...\n"
+    fi
+else
+    printf_yellow "Find and install our applications.\n"
+fi
+
+TARGET="install"
+printf_yellow "\nSIRIUS packages will be installed in: "
+if [ "$DEVELOP" == "yes" ]
+then
+    TARGET="develop-install"
+    printf_green "Develop mode.\n\n"
+else
+    printf_green "Install mode.\n\n"
+fi
+
+printf_yellow_clear "Installing SIRIUS Control System related packages.\n"
+clone_or_find mathphys lnls-fac && make $TARGET
+clone_or_find dev-packages lnls-sirius && cd siriuspy && \
+    make $TARGET
+clone_or_find hla lnls-sirius && cd pyqt-apps && make $TARGET
+clone_or_find hlafac lnls-fac && make $TARGET
+if [ "$INST_SIM" == "yes" ]
+then
+    printf_yellow_clear "Installing accelerators simulation packages.\n"
+    clone_or_find lnls lnls-fac && make $TARGET
+    clone_or_find trackcpp lnls-fac && make clean && \
+        make install-cpp 2>/dev/null && make $TARGET-py 2>/dev/null
+    clone_or_find pyaccel lnls-fac && make $TARGET
+    clone_or_find pymodels lnls-fac && make $TARGET
+    clone_or_find apsuite lnls-fac && make $TARGET
+fi
+if [ "$INST_COL" == "yes" ]
+then
+    printf_yellow_clear "Installing collective effects simulation packages.\n"
+    clone_or_find collective_effects lnls-fac && cd cppcolleff && \
+        make clean && make install-cpp 2>/dev/null && \
+        make $TARGET-py 2>/dev/null && \
+        cd ../pycolleff && make $TARGET
+fi
+if [ "$INST_IOC" == "yes" ]
+then
+    printf_yellow_clear "Installing SIRIUS IOCs related packages.\n"
+    clone_or_find eth-bridge-pru-serial485 lnls-sirius && cd client && \
+        pip install --no-deps -e ./
+    clone_or_find machine-applications lnls-sirius && \
+        make $TARGET
+fi
+if [ "$INST_IMA" == "yes" ]
+then
+    printf_yellow_clear "Installing magnets simulation packages.\n"
+    clone_or_find fieldmaptrack lnls-fac && make $TARGET
+    clone_or_find Radia lnls-sirius && make install 2>/dev/null
+    clone_or_find idanalysis lnls-fac && make $TARGET
+    clone_or_find insertion-devices lnls-ima && pip install -e ./
```

Comment:
For this one we need to "remove" the -e option when TARGET=install

Reply:
maybe would be better to adapt the Makefile of this repo, don't you think?

Reply:
I've done a PR lnls-ima/insertion-devices#22 adding a Makefile.

Reply:
I ran the test in a container with `./sirius-script-mamba-env-create.bash -b insertion-devices:add-makefile` and it worked fine!